sdk/dotnet/Pulumi/Core/OutputUtilities.cs

```diff
@@ -11,6 +12,27 @@ namespace Pulumi.Utilities
     /// </summary>
     public static class OutputUtilities
     {
+        /// <summary>
+        /// Create an unknown with the given value.
+        /// Note: generally, this should never be used since an unknown never resolves during preview.
```

Comment:
Since we're repeating the warnings perhaps we can lift them to the class level?

Reply:
Done